[2022-10-26 13:15:03,103] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag.fetch_file scheduled__2022-10-26T13:00:00+00:00 [queued]>
[2022-10-26 13:15:03,111] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag.fetch_file scheduled__2022-10-26T13:00:00+00:00 [queued]>
[2022-10-26 13:15:03,112] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-10-26 13:15:03,112] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-10-26 13:15:03,112] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-10-26 13:15:03,126] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): fetch_file> on 2022-10-26 13:00:00+00:00
[2022-10-26 13:15:03,128] {standard_task_runner.py:52} INFO - Started process 3434 to run task
[2022-10-26 13:15:03,131] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dag', 'fetch_file', 'scheduled__2022-10-26T13:00:00+00:00', '--job-id', '120', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpsnne90d0', '--error-file', '/tmp/tmpec3v7nso']
[2022-10-26 13:15:03,132] {standard_task_runner.py:80} INFO - Job 120: Subtask fetch_file
[2022-10-26 13:15:03,192] {task_command.py:369} INFO - Running <TaskInstance: dag.fetch_file scheduled__2022-10-26T13:00:00+00:00 [running]> on host cf01c04ee00d
[2022-10-26 13:15:03,262] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dag
AIRFLOW_CTX_TASK_ID=fetch_file
AIRFLOW_CTX_EXECUTION_DATE=2022-10-26T13:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-10-26T13:00:00+00:00
[2022-10-26 13:15:05,374] {python.py:173} INFO - Done. Returned value was: None
[2022-10-26 13:15:05,389] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=dag, task_id=fetch_file, execution_date=20221026T130000, start_date=20221026T131503, end_date=20221026T131505
[2022-10-26 13:15:05,515] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-10-26 13:15:05,552] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
